src/countervalues/react.js

```diff
@@ -91,18 +93,23 @@ export const Countervalues = ({
     );
   }, [pending, state, userSettings, triggerPoll]);
 
-  // trigger poll only when userSettings is not exactly the same
+  // trigger poll for the frist render and when userSettings is not exactly the same
+  const isFirstRender = useRef(true);
   const userSettingsRef = useRef(userSettings);
   useEffect(() => {
-    if (
-      JSON.stringify(userSettings) === JSON.stringify(userSettingsRef.current)
-    )
-      return;
+    if (isFirstRender) {
+      isFirstRender.current = false;
+    } else {
+      if (
+        JSON.stringify(userSettings) === JSON.stringify(userSettingsRef.current)
```

Comment:
(LOW) I feel this is overkill to stringify each time, I wonder if there is a better way, but that was for sure the most MVP one.